numba_dppy/compiler.py

```diff
@@ -455,7 +456,7 @@ def _pack_argument(self, ty, val, sycl_queue, device_arr, access_type):
             # we get the date back to host if have created a
```

Comment:
s/date/data

Reply:
The comment should overall be updated. I think it is carry over from when we did have a `device_array`. Now that there is no such type, the comment should be cleaned up.

Stylistic nitpick: Comments can be 72 char long per PEP-8.